src/main/java/cohen/wordle/WordleGame.java

```diff
@@ -0,0 +1,71 @@
+package cohen.wordle;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Random;
+
+public class WordleGame
+{
+    private final WordleDictionary dictionary = new WordleDictionary();
+    private final Random random = new Random();
+    String actualWord;
+
+    public WordleGame() throws IOException
+    {
+        do
+        {
+            int picked = random.nextInt(167963);
+            this.actualWord = dictionary.words.get(picked);
+        }
+        while(actualWord.length() != 5);
+    }
+
+    public String getActualWord ()
+    {
+        return actualWord;
+    }
+
+    public CharStatus[] guess (String guess)
+    {
+        guess = guess.toUpperCase();
+        CharStatus[] results = new CharStatus[]
+                {
+                        CharStatus.NotFound,
+                        CharStatus.NotFound,
+                        CharStatus.NotFound,
+                        CharStatus.NotFound,
+                        CharStatus.NotFound,
+                };
+
+        if (dictionary.words.contains(guess) && guess.length() == 5)
```

Comment:
`dictionary.words.contains(guess)` isn't necessary and actually takes a while to check all 160k words.